DevOps-Pipeline-PHP/laravel-app/check_installations.sh

```diff
@@ -0,0 +1,136 @@
+#!/bin/bash
+
+# Check Java
+echo "Checking Java installation..."
+java -version
+if [ $? -eq 0 ]; then
+    echo "Java installed successfully."
+else
+    echo "Java installation failed."
+fi
+
+# Check Python
+echo "Checking Python installation..."
+python3.9 --version
```

Comment:
@amtawfik  
python3 --version

Change this only and I will merge it and add you to the contributors of this repository 🚀